backend/routes/authRoutes.js

```diff
@@ -0,0 +1,41 @@
+const express = require("express");
+const router = express.Router();
+const { login, signup } = require("./authServices.js");
+
+// POST /auth/login
+// @params username: string, pwd: string
+router.post("/login", async (req, res) => {
+  const username = req.body.username;
+  const pwd = req.body.pwd;
```

Comment:
this is the thing Rhys was mentioning already. Originally I was thinking it could break on login since the user docs use "password", but I was thinking about it more and it probably doesn't actually matter if we use pwd or password in the req body, because we aren't going to be passing that around at all. In fact if we use pwd we can make sure _not_ to use that data anywhere or hold  onto it at all.

Reply:
I changed all "pwd" to "password" to avoid any confusion.